### PR TITLE
Added new page for hindi version

### PR DESCRIPTION
**Pull request by jitendravyas**
We can keep editing this page

### CODE REVIEW

**Comment by jitendravyas** [comment]
and I don't know which url we will use for hindi version. I think hindi.html could be better file name

File: src/index-hindi.html [hunk 1]
```diff
@@ -0,0 +1,655 @@
...
+    </div>
+  </nav>
+  <div id="header">
+  </div>
+  <div class="container">
+    <div class="row">
+         <div class="col-xs-12 col-sm-6 col-md-9 col-lg-9">
+
+            <h1> इंटरनेट को बचायें (भाग 3)।
+            एक बार फिर से हमें TRAI <small>(भारतीय दूरसंचार नियामक प्राधिकरण)</small> को बताने की जरूरत है की हम इंटरनेट की निष्पक्षता के पक्ष में हैं (दिसंबर 2015)
```
Comment (miteshashar) [inline src/index-hindi.html]: Use `<abbr>` here?

Comment (miteshashar) [inline src/index-hindi.html]: `nispakshta ke paksh mein hain`. Doesnt that sound a little weird? I would probably say `nispakshta ke samarthan mein hain`

Comment (miteshashar) [inline src/index-hindi.html]: `TRAI ko bataane ki` sounds avengeful. What do you think about `TRAI ko suchit karne ki`?

File: src/index-hindi.html [hunk 2]
```diff
@@ -0,0 +1,655 @@
...
+    </h1>
+
+
+            <div class="alert alert-warning hidden" id="browser_warning" role="alert">
+              <span class="browser_name">This browser</span> is known to have problems submitting this form. Please use another browser if it doesn't work.
+            </div>
+            <noscript>You need a JavaScript-capable browser to submit this form.</noscript>
+            <div class="button-bar">
+             <p><strong>ट्राई ने एक और परामर्श पत्र जारी किया है जिस पर  वह जनता की राय जानना चाहती है  । इस बार, यह अंतर मूल्य निर्धारण के बारे में है - एयरटेल जीरो एवं फ्री बेसिक्स / Internet.org प्रकार की सेवाओं के बारे में</strong></p>
```
Comment (miteshashar) [inline src/index-hindi.html]: Doesn't `अंतर मूल्य निर्धारण` become too complicated for spoken Hindi? Can't we say something simple like `कीमत निर्धारण में असंगति ` which more people can understand?

File: src/index-hindi.html [hunk 3]
```diff
@@ -0,0 +1,655 @@
...
+            </div>
+            <button class="btn btn-info btn-lg sendResponseAuto" id="mainSend">ट्राई को अपना जवाब भेजें</button>
+              या <a href="#" class="chooseAnswers" data-toggle="collapse" data-target="#collapseContainer" id="editAnswers"> अपने जवाब को संपादित करकें भेजें </a> (ट्राई को  <strong>advisorfea1@trai.gov.in</strong> पर और हमें <strong class="bccAddressModal"></strong> पर बीसीसी करें)
```
Comment (miteshashar) [inline src/index-hindi.html]: `अपने जवाब को संपादित करकें भेजें` » `अपने जवाब को जरूरत के अनुसार बदलकर भेजें`

File: src/index-hindi.html [hunk 4]
```diff
@@ -0,0 +1,655 @@
...
+    <div class="row">
+         <div class="col-xs-12 col-sm-6 col-md-9 col-lg-9">
+
+            <h1> इंटरनेट को बचायें (भाग 3)।
+            एक बार फिर से हमें TRAI <small>(भारतीय दूरसंचार नियामक प्राधिकरण)</small> को बताने की जरूरत है की हम इंटरनेट की निष्पक्षता के पक्ष में हैं (दिसंबर 2015)
+    </h1>
+
+
+            <div class="alert alert-warning hidden" id="browser_warning" role="alert">
+              <span class="browser_name">This browser</span> is known to have problems submitting this form. Please use another browser if it doesn't work.
```
Comment (miteshashar) [inline src/index-hindi.html]: We should translate this too, for people using browsers we have known problems with.

File: src/index-hindi.html [hunk 5]
```diff
@@ -0,0 +1,655 @@
...
+</head>
+<body>
+  <nav class="navbar navbar-default navbar-fixed-top">
+    <div class="container">
+      <div class="navbar-header">
+        <a class="navbar-brand" href="#">#SaveTheInternet</a>
+        <a class="navbar-toggle btn btn-info hidden-md hidden-lg navBarSend sendResponse disabled" style="display:none;" id="mobileNavBarSend">Send</a>
+      </div>
+      <div class="nav navbar-nav navbar-right hidden-xs hidden-sm navBarSend" style="display:none;">
+        <button type="button" class="btn btn-info navbar-btn sendResponse disabled" id="navBarSend">Respond to TRAI now</button>
```
Comment (miteshashar) [inline src/index-hindi.html]: Like, https://github.com/netneutrality/savetheinternet.in/pull/146/files#diff-cfbc5c2e6c95e165f882116c822e785dR120
`ट्राई को अपना जवाब भेजें`

File: src/index-hindi.html [hunk 6]
```diff
@@ -0,0 +1,655 @@
...
+
+    </style>
+
+      <!-- Modal -->
+      <div class="modal fade" id="sendModal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel" aria-hidden="true">
+        <div class="modal-dialog">
+          <div class="modal-content">
+            <div class="modal-header">
+              <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
+              <h4 class="modal-title"> प्रतिक्रिया भेजें।</h4>
```
Comment (miteshashar) [inline src/index-hindi.html]: Like, https://github.com/netneutrality/savetheinternet.in/pull/146/files#diff-cfbc5c2e6c95e165f882116c822e785dR120
We should have `ट्राई को अपना जवाब भेजें`

File: src/index-hindi.html [hunk 7]
```diff
@@ -0,0 +1,655 @@
...
+            </div>
+            <div class="modal-body">
+              <label for="responseSubject">
+                <h4>विषय</h4>
+                <input type="text" id="responseSubject" readonly class="form-control"></label>
+              <h4>आप का जवाब</h4>
+              <textarea class="form-control" id="responseTextarea" readonly></textarea>
+              <h4 id="copy-instruction">ऊपर लिखा हुआ टेक्स्ट कॉपी करें<span class="copy-short"> by Right-click->Copy (or <span class="control-str">Ctrl</span>+C)</span> <button id="copyBtn" class="btn btn-primary">हो गया</button></h4>
```
Comment (miteshashar) [inline src/index-hindi.html]: Here & in the next line, we could use the word `लेख` instead of `टेक्स्ट`

File: src/index-hindi.html [hunk 8]
```diff
@@ -0,0 +1,655 @@
...
+              <div id="response-copied"><h4>एक ईमेल लिखें, फिर उसमें कॉपी किया हुआ टेक्स्ट<span class="copy-short"> (<span class="control-str">Ctrl</span>+V)</span> पेस्ट करें</h4>
+              <p class="button-bar">
+                <a href="#" target="_blank" class="btn btn-primary btn" id="gmailSend">Gmail</a>
+                <a href="#" target="_blank" class="btn btn-primary btn" id="yahooSend">Yahoo</a>
+                <a href="#" target="_blank" class="btn btn-primary btn" id="otherSend">Other</a>
+              </p>
+                <p>आप ईमेल भेजने से पहले संपादित कर सकेंगे</p>
```
Comment (miteshashar) [inline src/index-hindi.html]: How about `padh kar apne anusar badlav kar sakenge`

File: src/index-hindi.html [hunk 9]
```diff
@@ -0,0 +1,655 @@
...
+                </div>
+                <p>ईमेल <strong><span class="text-info">advisorfea1@trai.gov.in</span></strong><br />को भेजा जाना चाहिए
+                और कृपया हमारे ईमेल
+                 <strong><span class="text-info" id="bccAddressModal">netneutralityindia@gmail.com</span></strong> को BCC में रखें|<br />
+                हम आपकी प्रतिक्रिया ट्राई ने स्वीकार किया है कि यह सुनिश्चित करने की कोशिश करेंगे।</p>
```
Comment (miteshashar) [inline src/index-hindi.html]: This seems to be grammatically incorrect. `Ham aapki pratikriya TRAI ne svikar kiya hai ki yah…`

File: src/index-hindi.html (hunk 5 above)
Comment (jitendravyas) [inline src/index-hindi.html]: This was not visible on page so I did not change here

File: src/index-hindi.html (hunk 7 above)
Comment (jitendravyas) [inline src/index-hindi.html]: Lekh = Essay but this is a letter

Comment (miteshashar) [inline src/index-hindi.html]: Lekh is also = Paragraphs. A lot of hindi speaking users might not understand `text` na.

File: src/index-hindi.html (hunk 5 above)
Comment (miteshashar) [inline src/index-hindi.html]: This is visible in the workflow at certain times.

**Pull request merged**